spec/factories/incident.rb

```diff
@@ -1,31 +1,10 @@
 FactoryGirl.define do
-  factory :incident, class: Incident do
-    transient do
-      num_civilians 1
-      num_officers  1
-    end
-
+  factory :incident do
+    status 'in_review'
     user { User.find_by_user_id(build(:dummy_user).user_id) || create(:dummy_user) }
-
-    after(:create) do |incident, e|  # e is the FactoryGirl evaluator (has access to transient attributes).
-      incident.screener = create :screener
-
-      incident.general_info = create(:general_info) do |g|
-        g.update_attributes(
-          num_involved_civilians: e.num_civilians,
-          num_involved_officers: e.num_officers
-        )
-      end
-
-      e.num_civilians.times do
```

Comment:
Removing all of this because there is no need to manually define it.  Factory girl will build the associations.  If you go into rails console and run FactoryGirl.build(:incident), you will see that all the associations also get built.